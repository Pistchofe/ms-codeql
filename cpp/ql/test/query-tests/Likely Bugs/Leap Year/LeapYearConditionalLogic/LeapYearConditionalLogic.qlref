Likely Bugs/Leap Year/LeapYearConditionalLogic.ql
