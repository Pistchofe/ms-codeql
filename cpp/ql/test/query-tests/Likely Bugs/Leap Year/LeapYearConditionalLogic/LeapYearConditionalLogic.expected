| LeapYearConditionalLogic.cpp:171:5:177:5 | if (...) ...  | Leap Year conditional statement may have untested code paths |
